### PR TITLE
Prep webdev for publishing

### PR DESCRIPTION
**Pull request by annagrin**
Depend on latest dwds, fix typo

### CODE REVIEW

File: webdev/pubspec.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ dependencies:
   browser_launcher: ^1.0.0
   crypto: ^3.0.0
   dds: ^2.0.0
-  dwds: ^11.1.1
+  dwds: ^11.1.2
```
Comment (grouma) [inline webdev/pubspec.yaml:21]: I think the min SDK version needs to be ">=2.14.0" given this constraint.

**Review by grouma — APPROVED** [review]

File: webdev/pubspec.yaml (hunk 1 above)
Comment (annagrin) [reply webdev/pubspec.yaml:21]: I'll update the dwds version to 11.2.0, but I don't see anything in the change logs of dwds that requires 2.14.0 - am I missing something? I could update the min to 2.13.0 though now that we have a stable version.

Comment (annagrin) [reply webdev/pubspec.yaml]: I verified that all e2e tests pass with 2.13.4.

Comment (grouma) [reply webdev/pubspec.yaml]: `package:dwds` depends on `>=2.13.0-144.0.dev`. We prefer to have `webdev`'s min SDK be a stable version. Right now it's min SDK is `>=2.13.0` so we should bump it to `>=2.13.4`.

Comment (annagrin) [reply webdev/pubspec.yaml]: I will publish another dwds vesrion before updating this.

Comment (annagrin) [reply webdev/pubspec.yaml]: updated dwds to have correct constraints, done!

**Review by natebosch — APPROVED** [review]

**Pull request merged**